discord_quote/discord_quote/discord_quote.py

```diff
@@ -163,23 +225,20 @@ def misquote(ctx , target : discord.User):
         #    user = target
         #else:
         #    user = ctx.message.server.get_member(target)
-       
-        yield from bot.send_message(ctx.message.author,
-                                    ('What would you like to be '
-                                     + ' misattributed to ' 
-                                     + user.name + '?'))
+
+        yield from bot.send_message(ctx.message.author, ('What would you like to be ' + ' misattributed to ' + user.name + '?'))
```

Comment:
we should probably refactor these as f strings later,

Reply:
We need to be careful not to have fstrings anywhere close to the user input (e.g., `clean_content` or `content`), but otherwise, sure.

- cz